Taskfile.yml

```diff
@@ -22,20 +22,12 @@ tasks:
     platforms: [darwin]
     vars:
       BINARY: terraform-provider-opslevel_{{.VERSION_DYNAMIC}}
-      BINARY_PATH_BEFORE_VERSION: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel"
-      OS_ARCH:
-        sh: echo $(go env GOOS)_$(go env GOARCH)
-      VERSION_FROM_BINARY:
-        sh: echo {{.BINARY}} | cut -d '_' -f 2
+      LOCATION: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel/{{.VERSION_DYNAMIC}}/{{OS}}_{{ARCH}}"
     cmds:
       - go build -o {{.BINARY}} || exit 1
       - chmod +x {{.BINARY}}
-      - mkdir -p {{.BINARY_PATH_BEFORE_VERSION}}/{{.VERSION_FROM_BINARY}}/{{.OS_ARCH}}
-      - mv {{.BINARY}} {{.BINARY_PATH_BEFORE_VERSION}}/{{.VERSION_FROM_BINARY}}/{{.OS_ARCH}}
-      - cmd: echo "Built terraform provider at:"
-        silent: true
-      - cmd: echo "{{.ROOT_DIR}}/{{.BINARY_PATH_BEFORE_VERSION}}/{{.VERSION_FROM_BINARY}}/{{.OS_ARCH}}/{{.BINARY}}"
-        silent: true
+      - mkdir -p {{.LOCATION}} || teue
```

Comment:
typo: `teue` --> `true`

Reply:
@davidbloss - this was failing on my machine without the `|| true` part but i'll try again

Reply:
ok this doesn't matter - i swear at one point it was failing for me but could have just been some of the other changes i was making to the path